src/lib/actions/updateBinary.js

```diff
@@ -7,26 +7,29 @@ const { Application } = require('../application.js');
 
 
 module.exports = {
+
   do: async (cmd) => {
     const cfg = config.read(cmd.config);
 
-    console.log(chalk.yellow('Updating binary...'));
+    console.log(chalk.yellow('Syncing platform...'));
     const platform = new Platform(cfg);
     let platformResult;
     try {
       platformResult = await platform.output();
     } catch (e) {
       console.log(chalk.red(`Could not get output from platform: ${e.message}`));
+      console.log("ERROR:" + e.toString());
```

Comment:
Not sure if `e.toString()` is different from `e.message` In any case, why not include it in the above `console.log` call?